plugins/modules/win_domain_computer_info.ps1

```diff
@@ -0,0 +1,98 @@
+#!powershell
+Set-StrictMode -Version 2.0
```

Comment:
Just for the future this isn't needed unless you really want it. I personally find strict mode a bit too strict and fights against standard pwsh-isms but it's ultimately up to you the module author what you want to do.